### DISCUSSION TITLE
Windows version strongswan ext_auth plugin

### DISCUSSION THREAD

**Discussion by psalty** [strongswan/strongswan#1568] [Q&A] [answered]
I saw ext_auth plug-in is also ported to windows.

https://docs.strongswan.org/docs/5.9/os/windows.html

What kind of external shell script does it support in this case? PowerShell?

**Comment by tobiasbrunner** [comment, marked as answer]
It uses `cmd.exe /C <configured command/script>`.

**Reply by psalty** [reply]
so normal bash shell? I am not familiar with cmd shell though.

**Reply by tobiasbrunner** [reply]
No, that's the default/classic windows shell (i.e. the one that can execute .bat/.cmd scripts or .exes). If you need a bash shell, you have to start that first (i.e. the command configured for the _ext-auth_ plugin has to include that). I suppose similarly for PowerShell.
